app/models/credence_answer.rb

```diff
@@ -0,0 +1,9 @@
+class CredenceAnswer < ActiveRecord::Base
+  belongs_to :credence_question
+
+  def format
+    # Would be nice to format the text in bold.
+    gen = self.credence_question
```

Comment:
The background here is that CredenceQuestion used to be named CredenceQuestionGenerator, so gen was short for generator. We renamed the class without renaming the variables, which I realize is confusing.